src/components/course/EnrollButton.jsx

```diff
@@ -33,20 +35,20 @@ export default function EnrollButton() {
     pacingType,
     courseUuid,
   } = activeCourseRun;
-  const [isEnrollDisabled, setIsEnrollDisabled] = useState(false);
-  const [emailOptIn] = useState(false);
```

Comment:
We are dropping the B2C email opt in as we don't want to allow opting in to emails from the course pages for enterprise